packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -120,6 +178,32 @@
   width: 100%;
 }
 
+.card__checkbox {
+  position: absolute;
+  z-index: 1;
+  margin-top: var(--pharos-spacing-one-quarter-x);
+  margin-left: var(--pharos-spacing-one-quarter-x);
+}
+
+.card__selectable {
+  background-color: var(--pharos-color-marble-gray-94);
+}
+
+.card__selectable-card-hover {
+  background-color: var(--pharos-color-marble-gray-base);
+  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
+}
+
+.card__container-selected--error {
+  background-color: var(--pharos-color-white);
+  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
+}
+
+.card__container-selectable-hover--error {
+  background-color: inherit;
+  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
+}
```

Comment:
This SCSS isn't quite following the existing BEM approach the project uses. Based on the way this is all being used I think something like the following is more in line, requiring similar changes in the DOM:

```scss
.card__link--image.card__link--selectable {
  background-color: var(--pharos-color-marble-gray-94);
}

.card--selectable.card--hover {
  background-color: var(--pharos-color-marble-gray-base);
  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
}

.card__container--error.card__container--selected {
  background-color: var(--pharos-color-white);
  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
}

.card__container--error.card__container--selectable.card__container--hover {
  background-color: inherit;
  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
}
```